Sources/AppStoreConnectClient/AppStoreConnectClient.swift

```diff
@@ -125,17 +125,24 @@ public struct AppStoreConnectClient {
     public func fetchBuilds(
         for app: Application,
         with query: BuildsQuery
-    ) async throws -> [Build] {
+    ) async throws -> [Build]? {
+        guard let querySort = query.sort else {
+            throw AppStoreConnectError.missingSortParameter(
+                errors: "\(String(describing: query.sort))"
+            )
+        }
+        guard let sort = Sort(rawValue: querySort.rawValue) else {
+            throw AppStoreConnectError.invalidSortParameter(
+                errors: "\(String(describing: Sort(rawValue: querySort.rawValue)))"
+            )
+        }
+        let sortPayload = query.convertSort(sort)
+        let fieldsPayload = query.convertFields([])
```

Comment:
It is not correct 'cause you have empty array

Reply:
Changed.